types/lib/amazon-pay.d.ts

```diff
@@ -0,0 +1,48 @@
+import { Emitter } from './emitter';
+
+export type AmazonPayOptions = {
+  /**
+   * 2 Digit Country Code
+   */
+  region: string;
+
+  /**
+   * Specify which Payment Gateway in Recurly must handle the payment.
+   */
+  gatewayCode?: string
+
+  /**
+   * Sets button to Sandbox environment
+   */
+  sandbox?: boolean;
+
+  merchantId?: string;
```

Comment:
The `merchant_id` is fetched from the API before emitting the `ready` event. I don't think that it can be passed in as an option.